tests/js/server/inlineMessageBundlingMiddleware-tests.js

```diff
@@ -407,15 +408,14 @@ fluid.defaults("fluid.tests.handlebars.inlineMessageBundlingMiddleware.caseHolde
         queryLocaleRequest: {
             type: "fluid.tests.handlebars.inlineMessageBundlingMiddleware.request",
             options: {
-                path: "/messages?locale=en-GB"
```

Comment:
This is odd, how did we end up swapping "path" for endpoint?
I believe the blessed new name for this is "pathname" https://nodejs.org/api/url.html#url_url_pathname but this would not work because it doesn't include the query parameters.
It appears that under the new model there is no longer an approved name for "The URL after the hostname" as shown in the diagram https://nodejs.org/api/url.html#url_url_strings_and_url_objects - so I have probably answered my own question, but please confirm if my understanding is correct

Reply:
After having to chase down usages of the `fluid-express` test request grades and similar patterns, I realised that my old `endpoint` pattern was not always necessary and where possible collapsed in favour of using `path` and `port`.